### DISCUSSION TITLE
Does Netmake sync the config file using Client ?

### DISCUSSION THREAD

**Discussion by ramzcode** [gravitl/netmaker#2625] [Q&A]
I saw in one of the document it states that it can sync the server changes to the Netmaker Wire-Guard client, is this a correct statement ?

If i modify client's peer or allowed_ip list on the UI, will it update on the client side automatically ? or manually we need to update the allowed_ip list like we do for default wireguard configurations.

I can't see any detailed info on the same. if yes, then Hope this will work for CE edition as well post the Lic conversion too.

thanks in advance

**Comment by mattkasun** [comment]
changes made to a peer via the UI (or nmctl) will be automatically propogated to all netclients.

any existing ext_client configurations would need to be regenerated and redistributed.
